src/forms/DateTimeLocalInput.jsx

```diff
@@ -71,7 +75,7 @@ DateTimeLocalInput.propTypes = {
 		PropTypes.string,
 		PropTypes.element,
 	]),
-	onChangeCallback: PropTypes.func
+	onChange: PropTypes.func, // provided by redux-form or by a wrapping component eg DateTimePicker
```

Comment:
after the conversation - do you think `onChange` should still be a required prop?